tests/unit/test_models/test_base_model_generate_julia_diffeq.py

```diff
@@ -0,0 +1,132 @@
+#
+# Tests for the base model class
+#
+import platform
+import unittest
+import pybamm
+
+have_julia = True  # pybamm.have_julia()
+if have_julia and platform.system() != "Windows":
+    from julia.api import Julia
+
+    Julia(compiled_modules=False)
+    from julia import Main
+
+    # load julia libraries required for evaluating the strings
+    Main.eval("using SparseArrays, LinearAlgebra")
+
+
+@unittest.skipIf(not have_julia, "Julia not installed")
+@unittest.skipIf(platform.system() == "Windows", "Skipped for Windows")
+class TestBaseModelGenerateJuliaDiffEq(unittest.TestCase):
```

Comment:
looks like most of these tests are not using `Main`, so can we turn them on for windows?